openshift-maven-plugin/plugin/src/main/resources/META-INF/jkube/profiles-default.yml

```diff
@@ -102,7 +104,7 @@
     includes:
     - spring-boot
     - wildfly-swarm
-    - thorntail-v2
+    - wildfly-jar
```

Comment:
Is the substitution/replacement intended?

Reply:
Was not intended, thank-you for catching it!